kolibri/plugins/html5_viewer/options.py

```diff
@@ -0,0 +1,42 @@
+# Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox
+allowable_sandbox_tokens = set(
+    [
+        "allow-downloads-without-user-activation",
+        "allow-forms",
+        "allow-modals",
+        "allow-orientation-lock",
+        "allow-pointer-lock",
+        "allow-popups",
+        "allow-popups-to-escape-sandbox",
+        "allow-presentation",
+        "allow-same-origin",
+        "allow-scripts",
+        "allow-storage-access-by-user-activation ",
+        "allow-top-navigation",
+        "allow-top-navigation-by-user-activation",
+    ]
+)
+
+
+def clean_sandbox(sandbox_string):
+    """
+    Clean up sandbox string to ensure it only contains valid items.
+    """
+    sandbox_tokens = [
+        token
+        for token in sandbox_string.split(" ")
+        if token in allowable_sandbox_tokens
+    ]
+    return " ".join(sandbox_tokens)
+
+
+option_spec = {
+    "HTML5": {
+        "SANDBOX": {
+            "type": "string",
+            "default": "allow-scripts",
+            "envvars": ("KOLIBRI_HTML5_SANDBOX",),
+            "clean": clean_sandbox,
+        }
+    }
+}
```

Comment:
Rather than a default, should we maybe always add "allow-scripts" back in on top of whatever they specify? Otherwise Hashi will break anyway, right?

Reply:
My vote would be to leave the options fully customizable, just because it gives the user full control. While I can't think of a reason a user might want to block scripts from running, I think if someone did want that behavior they should be able to do so. (And blocking Hashi in that case would be expected behavior - I suppose we could avoid injecting Hashi in that case even.)